.pre-commit-config.yaml

```diff
@@ -51,6 +51,7 @@ repos:
         entry: tofu fmt -diff -check
         language: system
         files: ^packages/infrastructure/
+        exclude: \.sh$
```

Comment:
added to resolve 

Tofu Format..............................................................Failed
- hook id: tf-fmt
- exit code: 2

╷
│ Error: Only .tf, .tfvars, and .tftest.hcl files can be processed with tofu fmt
│ 
│ 
╵